src/libsystemd/sd-bus/bus-error.c

```diff
@@ -78,8 +76,10 @@ static int bus_error_name_to_errno(const char *name) {
                 return r;
         }
 
+        const sd_bus_error_map *m;
```

Comment:
i'd probably nove this into the for() loop below where it is used for the first time, and then declare it a 2nd type for the ALIGN_PTR() assignment further down.